src/containers/BooksLists.js

```diff
@@ -1,16 +1,37 @@
 import { useSelector, useDispatch } from 'react-redux';
-import { removeBook } from '../actions/actions';
+import { removeBook, changeFilter } from '../actions/actions';
 import Book from '../components/Book';
+import CategoryFilter from '../components/CategoryFilter';
 
 function BooksList() {
-  const list = useSelector((state) => state);
+  const bookListState = useSelector((state) => state.books);
+  const filterState = useSelector((state) => state.filter);
   const dispatch = useDispatch();
 
   function handleRemoveBook(e) {
```

Comment:
[Optional]
- [ ] Let us use ES6 arrow syntax instead of the function keyword. It is important to maintain consistency in our code base. We should not mix up ES5 and ES6 syntaxes. Let us use ES6 syntax :slightly_smiling_face:

Reply:
This change has been made well